Sources/Benchmarks/NonBlockingThreadPool.swift

```diff
@@ -39,7 +39,7 @@ let nonBlockingThreadPool = BenchmarkSuite(name: "NonBlockingThreadPool") { suit
       { pool.join({ pool.join({}, {}) }, { pool.join({}, {}) }) })
   }
 
-  suite.benchmark("join, four levels, three on thread pool thread", settings: .iterations(1000)) {
+  suite.benchmark("join, four levels, three on thread pool thread") {
```

Comment:
Nice to see auto-detection working well enough not to require manual settings. 😃

Reply:
Yeah, it's definitely quite nice. I do think that for some benchmarks, we can probably tune it to finish faster than 1s. As I add more and more benchmarks, it's nice to keep turnaround times fast. For now, I'm filtering down the benchmarks that I think are relevant. In the future, if there is a nice way to automatically detect that we're not getting much more signal, being able to terminate early would be nice. (That said, there's a pretty large computed standard deviation in my benchmark runs, so perhaps this wouldn't even help in my case...)

Reply:
Oh, and I did run into the infinite loop bug that you already fixed, so thank you for fixing that one so quickly!

Reply:
Note: this is not an important feature to work on now, however. :-)

Reply:
Why not change `--min-time` for iterative development to be `0.1` (or less if desired) ? Setting defaults too low in terms of iterations or too little time can give a false impression of statistical significance you are going to get out of the box.

Reply:
That's a great idea. I'll do that going forward. Thanks!